Tests/ServiceTierTests/LoginServiceTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Common.Exceptions;
+using Common.Model;
+using gtdtimer.Services;
+using Microsoft.AspNet.Identity;
+using Moq;
+using NUnit.Framework;
+using Timer.DAL.Timer.DAL.Entities;
+using Timer.DAL.Timer.DAL.Repositories;
+using Timer.DAL.Timer.DAL.UnitOfWork;
+
+namespace ServiceTierTests
+{
+    class LoginServiceTests
+    {
+        private Mock<IUnitOfWork> unitOfWork;
+        private Mock<IUserEmailStore<User,int>> userRepository;
+        private Mock<ApplicationUserManager> userManager;
+        private LogInService subject;
+        const string TokenPartTest = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9";
+
+        [SetUp]
+        public void Setup()
+        {
+            unitOfWork = new Mock<IUnitOfWork>();
+            userRepository = new Mock<IUserEmailStore<User, int>>();
+            userManager = new Mock<ApplicationUserManager>(userRepository.Object);
```

Comment:
hmm.. probably better to use interface instead of real class. This is minor issue tho

Reply:
I can`t use in this case IApplicationUserManager instead ApplicationUserManager, if I replace real class to interface it thow exception

Reply:
what about using base UserManager class?

Reply:
I also can`t use UserManager, can`t convert from UserManager to IAplicationUserManager. We create ApplicationUserManager that is inheritenced from IAplicationUserManager and use this class in UnitOfWork
![image](https://user-images.githubusercontent.com/37620894/50475349-17938400-09cd-11e9-9716-9a13de785d08.png)

Reply:
Maybe just use ApplicationUserManager?

Reply:
and what exception is thrown if you use IAppUserManager interface?

Reply:
Everything is Ok. I change the mock type to IApplicationUserManager. Maybe I a little bit change IAplicationUserManager and Startup class, when I try to fix saving password hash, that is why it doesn`t allow change ApplicationUserManager or maybe something another...

Reply:
awesome